app/models/metasploit/credential/ntlm_hash.rb

```diff
@@ -1,5 +1,22 @@
 require 'net/ntlm'
 
+# TODO: Revert once available in rubyntlm
+# https://github.com/WinRb/rubyntlm/pull/51
```

Comment:
Temporarily applying the patch from https://github.com/WinRb/rubyntlm/pull/51 until it's available as a release

Reply:
cc @jmartin-r7 Are we happy with this patch being here? I envision rubyntlm will take a long time to be fixed properly upstream

The unit tests caught this edgecase for us

https://github.com/rapid7/metasploit-credential/blob/ac481e8b05b8bcd878d8c88f02676be3eed4e1bf/spec/models/metasploit/credential/ntlm_hash_spec.rb#L354

Reply:
This seems reasonable.